reference_simulations/first_generation/telescope_config/mwa_config_uvbeam.yaml

```diff
@@ -0,0 +1,8 @@
+beam_paths:
+  0: !UVBeam
+      filename: http://ws.mwatelescope.org/static/mwa_full_embedded_element_pattern.h5
+      pixels_per_deg: 1
+      file_type: mwa_beam
```

Comment:
I'd suggest adding `mount_type: phased` to avoid some deprecation warnings that will turn into errors in the future.